WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -126,6 +126,10 @@ class MySiteViewModel @Inject constructor(
     private val _activeTaskPosition = MutableLiveData<Pair<QuickStartTask, Int>>()
     private val _onShowSwipeRefreshLayout = MutableLiveData<Event<Boolean>>()
 
+    /* Capture and track the site selected event so we can circumvent refreshing sources on resume
+       as they're already built on site select. */
+    private var _isSiteSelected = false
```

Comment:
Minor (🔍): The `_` backing field prefix is not needed here, consider removing it to avoid confusion about it being a live data field.

Reply:
In 351d190